kokoro/gcp_ubuntu/cmake/build_kokoro.sh

```diff
@@ -31,7 +31,8 @@ docker run \
   --volume "${WORKDIR?}:${WORKDIR?}" \
```

Comment:
Working on setting up a build for this. I think we want to have the GPU testing in a separate job. We're much more resource constrained there. Could you put this under cmake/linux/x86-turing instead?